roles/testnode/tasks/pip.yml

```diff
@@ -1,16 +1,18 @@
 ---
+# python-pip is installed from epel repo which doesn't exist on fedora.
+# package is installed during kickstart instead.
```

Comment:
python-pip is not part of default installation on Fedora.  It gets installed during/after a kickstart when cobbler runs ansible on the host.